File: src/Toto.CineOrg.ServiceModel/Movie.cs
using System;

namespace Toto.CineOrg.ServiceModel
{
    // ReSharper disable UnusedAutoPropertyAccessor.Global
    public class Movie
    {
        public Guid Id { get; set; }

        public string Title { get; set; } = null!;

        public string Description { get; set; } = null!;
        
        public string Genre { get; set; } = null!;
        
        public int YearReleased { get; set; }
    }
}

File: src/Toto.CineOrg.Ui.Angular/projects/admin-portal/src/app/models/movie.ts
export interface Movie {
  id: string;
  title: string;
  genre: string;
  yearReleased: number;
  description: string;
}

export interface MovieForCreate {
  title: string;
  genre: string;
  yearReleased: number;
  description: string;
}

export interface MovieForUpdate {
  title: string;
  genre: string;
  yearReleased: number;
  description: string;
}
